.pre-commit-config.yaml

```diff
@@ -58,17 +58,6 @@ repos:
         types: [file, yaml]
         stages: [commit]
 
-  - repo: local
-    hooks:
-      - id: mypy
```

Comment:
Is there a reason this has been removed in this PR?

Reply:
It was never part of the CI anyway and it was not respected. Currently, there are multiple mypy errors that emerge every time the hookds get activated and it is a lot of work to go around and fix em.